packages/shared/styles/components/SfColor.scss

```diff
@@ -0,0 +1,44 @@
+@import "../variables";
+
+$color__width: 2.5rem !default;
+$color__height: 2.5rem !default;
```

Comment:
maybe it can be just one property? radius or sth like that

Reply:
Thought of that but haven't found any applicable property on MDN docs.